### PR TITLE
Added Elemental Resist Status Icon

### PR DESCRIPTION
**Pull request by Emistry**
### Pull Request Prelude

- [x] I have followed [proper Hercules code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed

- show status icon for elemental resist status change (ItemID: `12118, 12119, 12120 and 12121`).

img

**Issues addressed:** 
https://github.com/rathena/rathena/issues/1898

[cont]: https://github.com/HerculesWS/Hercules/blob/master/CONTRIBUTING.md
[code]: https://github.com/HerculesWS/Hercules/wiki/Coding-Style

### CODE REVIEW

**Comment by HerculesWSAPI** [comment]
This change is [Reviewable](https://reviewable.io/reviews/herculesws/hercules/2516)

File: src/map/status.c [hunk 1]
```diff
@@ -8847,7 +8847,9 @@ static int status_change_start(struct block_list *src, struct block_list *bl, en
 			case SC_KAIZEL:
 				val2 = 10*val1; //% of life to be revived with
 				break;
-				// case SC_ARMORPROPERTY:
+			case SC_ARMORPROPERTY:
+				clif->status_change(bl, val1 > 0 ? SI_RESIST_PROPERTY_WATER : val2 > 0 ? SI_RESIST_PROPERTY_GROUND : val3 > 0 ? SI_RESIST_PROPERTY_FIRE : val4 > 0 ? SI_RESIST_PROPERTY_WIND : SI_BLANK, 1, tick, 0, 0, 0);
```
Comment (4144) [inline src/map/status.c]: here better to use brackets. because condition too complex and here can be mistake

Comment (Emistry) [reply src/map/status.c]: updated

**Comment by feltenc** [comment]
image
The status icon dont disappear. Git HASH 75ae0182f850e213d50832c979312aa560152628

**Pull request merged**